lib/util/cache/package/redis.ts

```diff
@@ -117,8 +117,13 @@ export async function init(
     pingInterval: 30000, // 30s
   };
   if (clusteredMode) {
+    const parsedUrl = new URL(rewrittenUrl);
     client = createCluster({
       rootNodes: [config],
+      defaults: {
```

Comment:
For the case with no username or password, I'd prefer there to be no defaults at all. Ie exact backwards compatibility